src/osd/scrubber_common.h

```diff
@@ -282,6 +283,17 @@ struct PgScrubBeListener {
   virtual bool is_waiting_for_unreadable_object() const = 0;
 };
 
+
+struct ScrubIoCounterSet {
+  osd_perf_counters_t getattr_cnt; ///< get_attr calls count
+  osd_perf_counters_t stats_cnt;  ///< stats calls count
+  osd_perf_counters_t read_cnt;   ///< read calls count
+  osd_perf_counters_t read_bytes;  ///< total bytes read
+  osd_perf_counters_t omapgetheader_cnt; ///< omap get header calls count
+  osd_perf_counters_t omapgetheader_bytes;  ///< bytes read by omap get header
+  osd_perf_counters_t omapget_cnt;  ///< omap get calls count
+  osd_perf_counters_t omapget_bytes;  ///< total bytes read by omap get
```

Comment:
Why use this intermediate rather than just adjusting the perf counters directly?

Reply:
I am using separate "as primary" vs "as replica" counters sets. And - as there is no guarantee that OSD labeled counters will be exportable in 'T' (after a sobering discussion with Yaartit) - I plan to move the existing 4 sets of existing scrub counters, now implemented as 2X2 labels and one set, into this new mechanism.

Reply:
I misunderstood, these are indices rather than counters.  Makes sense.